guides/doc-Administering_Red_Hat_Satellite/topics/Starting_and_Stopping_Satellite.adoc

```diff
@@ -1,6 +1,10 @@
 [[chap-Administering-Starting_and_Stopping_Red_Hat_Satellite]]
 == Starting and Stopping {ProjectName}
 
+ifndef::satellite[]
```

Comment:
What about Orcharhino?

Reply:
What do you mean exactly?

Reply:
He meant that I hadn't taken them into consideration - whether they would use this or not. 
My plan was that I was going to ask @maximiliankolb  for a review to ensure I didn't cut them out. 
This PR is dead, and won't be resurrected, but appreciate your feedback!